test/tools/spec-runner/index.js

```diff
@@ -597,6 +598,18 @@ function maybeSession(operation, context) {
   );
 }
 
+function withoutCommandMonitoring(client, fn) {
```

Comment:
Now that the option to ignore the ping is passed through, do we need to do this anymore?

Reply:
Yeah, this is for the `listCollections`/`listIndexes` calls that are part of the `assert(Index|Collection)(Not)?Exists`. I admittedly don’t quite know if and how the unified test runner handles this.

Reply:
The unified runner handles it by doing the assertions on the utility client and not the client under test - so those events aren't monitored.

Reply:
Yeah, that makes sense – the runner here also has a `utilClient`, but it closes it again before the test is run, so I figure that’s not really a viable option here?

Reply:
Yeah I think it's fine to do it as is.